rwengine/src/objects/InstanceObject.cpp

```diff
@@ -43,20 +43,17 @@ InstanceObject::~InstanceObject()
 void InstanceObject::tick(float dt)
 {
 	if( dynamics && body ) {
-		if( body->body->isStaticObject() ) {
-			if( _enablePhysics ) {
+		if( _enablePhysics ) {
+			if( body->getBulletBody()->isStaticObject() ) {
 				// Apparently bodies must be removed and re-added if their mass changes.
-				engine->dynamicsWorld->removeRigidBody(body->body);
-				btVector3 inert;
-				body->body->getCollisionShape()->calculateLocalInertia(dynamics->mass, inert);
-				body->body->setMassProps(dynamics->mass, inert);
-				engine->dynamicsWorld->addRigidBody(body->body);
+				body->changeMass(dynamics->mass);
 			}
 		}
 		
 		_updateLastTransform();
 
-		auto _bws = body->body->getWorldTransform().getOrigin();
+#warning replace with position from motionstate
```

Comment:
That's quite an odd compile-time warning.
Isn't this supposed to be a `/// @todo` + issue instead?